src/java/picard/analysis/CollectWgsMetricsFromQuerySorted.java

```diff
@@ -158,6 +158,8 @@ private void calculateMetricsForRead(final QuerySortedReadPairIteratorUtil.ReadP
         metrics.metrics.TOTAL_BASES += totalReadBases;
         if (isPaired) metrics.metrics.TOTAL_READ_PAIRS++;
 
+        // Note that CollectWgsMetrics does NOT count paired reads that are both unmapped in the PCT_EXC_UNPAIRED, but we do so here
+        //   because this tool isn't a locus iterator and we need to ensure that our passing base numbers are accurate in the end
```

Comment:
The point of this note is lost, I think. can you specify more clearly what metric you are trying to make accurate and what metric will be different?